{
    "schema_version": "1.6.0",
    "id": "OESA-2022-1809",
    "modified": "2022-08-05T00:00:00Z",
    "published": "2022-08-05T00:00:00Z",
    "withdrawn": "",
    "aliases": null,
    "related": [
        "CVE-2021-46823"
    ],
    "summary": "python-ldap security update",
    "details": "python-ldap provides an object-oriented API for working with LDAP within Python programs.  It allows access to LDAP directory servers by wrapping the OpenLDAP 2.x libraries, and contains modules for other LDAP-related tasks (including processing LDIF, LDAPURLs, LDAPv3 schema, etc.).\r\n\r\nSecurity Fix(es):\r\n\r\npython-ldap before 3.4.0 is vulnerable to a denial of service when ldap.schema is used for untrusted schema definitions, because of a regular expression denial of service (ReDoS) flaw in the LDAP schema parser. By sending crafted regex input, a remote authenticated attacker could exploit this vulnerability to cause a denial of service condition.(CVE-2021-46823)",
    "severity": null,
    "affected": [
        {
            "package": {
                "ecosystem": "openEuler-20.03-LTS-SP3",
                "name": "python-ldap",
                "purl": "pkg:rpm/python-ldap@3.1.0-4.oe1"
            },
            "severity": null,
            "ranges": [
                {
                    "type": "ECOSYSTEM",
                    "repo": "",
                    "events": [
                        {
                            "introduced": "0",
                            "fixed": "3.1.0-4.oe1",
                            "last_affected": "",
                            "limit": ""
                        }
                    ],
                    "database_specific": null
                }
            ],
            "versions": null,
            "ecosystem_specific": {
                "aarch64": [
                    "python3-ldap-3.1.0-4.oe1.aarch64.rpm"
                ],
                "noarch": [
                    "python-ldap-help-3.1.0-4.oe1.noarch.rpm"
                ],
                "src": [
                    "python-ldap-3.1.0-4.oe1.src.rpm"
                ],
                "x86_64": [
                    "python3-ldap-3.1.0-4.oe1.x86_64.rpm"
                ]
            },
            "database_specific": null
        }
    ],
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-46823"
        }
    ],
    "credits": null,
    "database_specific": {
        "severity": "Medium"
    }
}